libs/permission-checker/src/PermissionChecker.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PermissionChecker;
+
+use Keboola\PermissionChecker\Exception\PermissionDeniedException;
+
+class PermissionChecker
```

Comment:
Naming is hard, ale v tomhle místě se ptám, proč to není PermissionChecker implements PermissionCheckerInterface a proč checkPermissions nepřijmá PermissionChecker ?

tak jednoduše bych řekl PermissionCheckerInterface -> PermissionCheckInterface
nebo PermissionRuleInterface